src/rgw/librgw.cc

```diff
@@ -241,7 +242,7 @@ namespace rgw {
     /* XXX and -then- stash req_state pointers everywhere they are needed */
     ret = req->init(rgw_env, &rados_ctx, io, s);
     if (ret < 0) {
-      dout(10) << "failed to initialize request" << dendl;
+      ldpp_dout(s, 10) << "failed to initialize request" << dendl;
```

Comment:
@cbodley should he be using `op` here instead of `s`?

Reply:
yes